src/stdatamodels/jwst/datamodels/schemas/mirimg_photom.schema.yaml

```diff
@@ -23,3 +23,15 @@ allOf:
         datatype: float32
       - name: uncertainty
         datatype: float32
+- type: object
+  properties:
+    correction_table:
```

Comment:
I know it's been said that a foolish consistency is the hobgoblin of little minds, but for the sake of consistency with the MRS time-dependent entries in the photom ref file, would it be a lot of work to change the title here to "timecoeff" and the fits_hdu to "TIMECOEFF"? That would of course require changes in the photom step code too, but I think it's a pretty simple substitution. The name "correction" is kind of generic and hence it's not obvious at first sight what kind of correction it is.

Reply:
I can do that and I have no particular opinion on the naming. So you are suggesting to change "Correction_table" to timecoeff"? What about the second name? Fyi, it is technically not the exact same correction the MRS does, but it is OK.

Reply:
Yes, they could both be named "timecoeff". The first refers to the name used in the data model and the 2nd is the value for the EXTVER header keyword in FITS files.

Reply:
Done both here and in the jwst pipeline PR.